### ISSUE TITLE
Cant join server anymore (Client 0 overflowed reliable channel.) Might be a problem with duping entities

### ISSUE THREAD

**Issue by MrCookie13** [rebel1324/NutScript#189]
Suddenly I cannot join my server anymore.
After changing some code I got kicked out of the server with the error
(Client 0 overflowed reliable channel.)
After this I restarted the server and this pops up in the console.

```
`Client "Mr_Cookie1311" connected ().
WCD: Initializing player Mr_Cookie1311.
Mr_Cookie1311 owns 2 vehicles
 
Loaded (11, 31, 46) for Mr_Cookie1311
Dropped Mr_Cookie1311 from server (Client 0 overflowed reliable channel.)
 
[ERROR] gamemodes/nutscript/gamemode/core/libs/sh_character.lua:749: Tried to use a NULL entity!
  1. steamName - [C]:-1
   2. Nick - gamemodes/nutscript/gamemode/core/libs/sh_character.lua:749
    3. unknown - RunString:158
 
Timer Failed! [Simple][@RunString (line 152)]
 
[ERROR] gamemodes/nutscript/gamemode/shared.lua:14: Tried to use a NULL entity!
  1. nutSteamID64 - [C]:-1
   2. SteamID64 - gamemodes/nutscript/gamemode/shared.lua:14
    3. unknown - RunString:145
 
Timer Failed! [Simple][@RunString (line 143)]`
```
After changing the map to a other map I can join the server again but changing back to the first map gives me the exact error as before

**Comment by rebel1324** [comment]
did you updated the framework

**Comment by MrCookie13** [comment]
No I didn't update the framework

Don't use beta 1.1

**Comment by MrCookie13** [comment]
Updated frame work still have duplicating entities.

**Comment by rebel1324** [comment]
still? did you cleaned up the vendors

**Comment by rebel1324** [comment]
do you lua-refresh very often?

**Comment by SleepyMode** [comment]
@MrCookie13

**Comment by rebel1324** [comment]
The duplicating bug is completely fixed in 1.1-beta. I'll find some way to get this thing in main 1.1 branch.
